src/Flickr/Server.php

```diff
@@ -22,7 +22,9 @@ public function urlTemporaryCredentials()
      */
     public function urlAuthorization()
     {
-        return 'https://www.flickr.com/services/oauth/authorize';
+        $perms = env('FLICKR_AUTHORIZATION_PERMS');
```

Comment:
Please use `additionalConfigKeys` and `getConfig` and update the readme

Reply:
Unfortunately, I do not see any documentation on utilizing this:

https://socialiteproviders.com/contribute/

Perhaps I missed where this is?

Additionally, I looked through some other providers using these methods and am failing to see documentation there that might provide some clues on exactly what you are wanting. I simply looked through numerous of the providers here and saw that `env` was used directly, so followed by example.